platforms/core-configuration/graph-serialization/src/main/kotlin/org/gradle/internal/serialize/graph/Codec.kt

```diff
@@ -149,6 +173,12 @@ interface IsolateOwner {
 }
 
 
+interface IsolateContextSource {
```

Comment:
🧩 This belongs to `:configuration-cache-base` since it's only a contract between `:configuration-cache` and `:core-serialization-codecs`